host.json

```diff
@@ -10,14 +10,18 @@
   },
   "extensionBundle": {
     "id": "Microsoft.Azure.Functions.ExtensionBundle",
-    "version": "[3.*, 4.0.0)"
+    "version": "[2.*, 4.0.0)"
   },
   "concurrency": {
     "dynamicConcurrencyEnabled": false,
     "snapshotPersistenceEnabled": false
   },
   "functionTimeout": "00:10:00",
   "extensions": {
+    "http": 
```

Comment:
So this will affect the other functions deployed within this app, right?

Can we double check that the call to `http-parser` function still works? There are instructions in the readme on how to run the other functions locally. At a minimum, we should make sure the `http-parser` function logs that it received a request.